README.md

```diff
@@ -70,8 +70,8 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- coming soon after aprroved ....
-- coming soon after approved ....
```

Comment:
- [ ] [OPTIONAL] All the changes in your project are not imported into the PR. It is hard to give a proper review in this sitatuin. Please avoid doing this.